### PR TITLE
Move repo_user/name parsing to webhook_json_parser

### PR DESCRIPTION
**Pull request by alexjfisher**

### CODE REVIEW

File: lib/parsers/webhook_json_parser.rb [hunk 1]
```diff
@@ -96,6 +98,24 @@ def deleted?
...
         end
       end
+
+      def repo_name
+        case @vcs
+        when 'github'
+          @data['repository']['name']
+        end
+        # TODO: Bitbucket, Stash/Bitbucket Server, Gitlab, TFS
+        nil
```
Comment (ekohl) [inline lib/parsers/webhook_json_parser.rb]: This should be in an ```else``` branch of the ```case``` statement. Same for ```repo_user``` below.

Comment (alexjfisher) [reply lib/parsers/webhook_json_parser.rb]: That's what I started with, but rubocop hates me. `Redundant 'else'-clause`

Comment (dhollinger) [reply lib/parsers/webhook_json_parser.rb]: Does anyone know what the purpose of these two methods originally was?

Comment (alexjfisher) [reply lib/parsers/webhook_json_parser.rb]: Sorry, it was obviously wrong before unless I added some `returns`.

**Pull request merged**